docker-compose/api-test.py

```diff
@@ -0,0 +1,79 @@
+import requests
+import unittest
+import random
+import string
+import json
+
+class WalletAPITest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.wallet_url = "http://localhost:7001"
+        cls.headers = {"Content-Type": "application/json" }
+        cls.name = ''.join(random.choices(string.ascii_letters + string.digits, k=8))
+        cls.email = f"{cls.name}@example.com"
```

Comment:
I'd rather randomize the email than the name

Reply:
The email already uses the randomized name for prefix, therefore i guess the email is randomized aswell, or do you have another idea ?

Reply:
oh, my bad